pwa/src/data/navMenuItems.tsx

```diff
@@ -96,6 +97,12 @@ export const mainMenuItems = () => {
       onClick: () => navigate("/mappings"),
       current: pathname.includes("/mappings"),
     },
+    {
+      label: "Templates",
+      icon: <FontAwesomeIcon icon={faFile} />,
+      onClick: () => navigate("/templates"),
+      current: pathname.includes("/templates"),
+    },
```

Comment:
I think we should move "Logs" to the bottom of this list, that way we have:

1. Dashboard
2. ...all the resources
3. Logs of everything